chain/vm/burn.go

```diff
@@ -76,7 +76,12 @@ func ComputeGasOutputs(gasUsed, gasLimit int64, baseFee, feeCap, gasPremium abi.
 		baseFeeToPay = feeCap
 		out.MinerPenalty = big.Mul(big.Sub(baseFee, feeCap), gasUsedBig)
 	}
-	out.BaseFeeBurn = big.Mul(baseFeeToPay, gasUsedBig)
+
+	// If burning is disabled, just skip computing the BaseFeeBurn. However,
```

Comment:
"network transaction fee" would be more descriptive than burn, here and for the parameter name.